gedditlive/GedditSDK
# Voucher Feature

# Changelog

- Added: Display voucher when user won a voucher.
- Added: Subscribe to winning voucher event.
- Added: Callback SDK client when user won a voucher.

# Testing

In order to test using this branch, you should edit the cocoapods repository used as a source for your Podfile:

1. Make sure that `~/.cocoapods/repos/gedditlive` exists
    - If not, Run `pod install` with the following Podfile:

```ruby
source 'https://github.com/gedditlive/GedditSDK.git'
target 'YOUR_TARGET' do
  use_frameworks!

  pod 'GedditCoreSDK', '1.1.4'
  pod 'GedditCommerceSDK', '1.5.0'
end
```

2. Switch to the `release-1.5.0` branch:

```sh
cd ~/.cocoapods/repos/gedditlive;
git checkout 1.5.0
```

3. Edit the podspec files to make `pod install` downloads the frameworks on `release-1.5.0` branch.

_~/.cocoapods/repos/gedditlive/GedditCommerceSDK/1.5.0/GedditCommerceSDK.podspec:_
```ruby
s.source = { :http => 'https://github.com/gedditlive/GedditSDK/blob/release-1.5.0/GedditCommerceSDK/1.5.0/GedditCommerceSDK.zip?raw=true'}
```

4. Make sure to clear your pod cache:

```sh
rm -rf ~/Library/Caches/CocoaPods/Pods/Release/GedditCommerceSDK
rm -rf ~/Library/Caches/CocoaPods/Pods/Release/GedditCoreSDK
```

5. Run `pod install` in your project
